templates/pages/books.html

```diff
@@ -96,9 +96,9 @@ <h3 style="clear: left">Programmer's Guide To Kotlin</h3>
 
             <h3 style="clear: left">Kotlin</h3>
             <div class="book-lang">tr</div>
-            <a href="https://www.kuramkitap.com/kotlin"><img
+            <a href="https://www.kitapyurdu.com/kitap/kotlin-/430490.html"><img
```

Comment:
@katia-energizer How did you search `kitapyurdu`? `kuramkitap.com` just moved to  konseykitap and url – [konseykitap.com/kotlin](https://www.konseykitap.com/kotlin)? Or not?